faqs/galaxy/dataset_metadata_missing.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Missing or Incomplete Dataset metadata 
+area: datasets   
+description:  Finding and Correcting Metadata
+layout: faq         
+---
+
+How to notice missing Dataset Metadata:
+- Dataset will not be downloaded when using the disk icon
+- Tools error when using a previously successfully used specific dataset
+- Tools error with a message that ends with: ``OSError: [Errno 2] No such file or directory.``
+
+Solution:
+Click on the the dataset's {% icon galaxy-pencil %} ✏️ to reach the _Edit Attributes_ forms and click either:
+- **Directly reset metadata** 
+  - Find the tab for the metadata you want to change, make the change, and save.
+- **Autodetect metadata**
```

Comment:
So here maybe end users would like to know if they should first reset metadata and then autodetect metadata, or should they choose which one to do? That would be very nice to clarify it. Many thanks.